kitchen.yml

```diff
@@ -1,6 +1,6 @@
 ---
 provisioner:
-  deprecations_as_errors: true
+  deprecations_as_errors: false
```

Comment:
We need to keep this on.

Reply:
This will bring up deprecation messages for upstream dependencies that we have no control over when they will get fixed, if they will get fixed. Leaving this in place will never get a fully passing CI run for all future pull requests.

Reply:
Below is a screenshot from one of the latest/passing tests in GH Actions. As you can see, the only deprecation messages are for upstream dependencies that we have no control over. This value set to `true` was previously causing the tests to FAIL for no other reason.

Reply:
![image](https://user-images.githubusercontent.com/1933549/118560886-0b83fa80-b71f-11eb-9881-289a5d033617.png)

Reply:
I know that the line cookbook should have those issues resolved by now. We shouldn't be using the chef-vault or ohai in this cookbook anymore as both of them either are or will be deprecated upstream and will not get fixes for Chef 17.

Reply:
Although chef-vault is built into Chef Infra Client, it does not have the data bag fallback from the `chef_vault_item` method that this cookbook supports. Until Chef Infra Client supports that in its implementation of chef-vault, we need to have the cookbook as a dependency. There have been some talk about removing the dependency on chef-vault altogether, but until a PR is submitted to do that, this cookbook will need to have the chef-vault cookbook as a dependency.

Reply:
Unless we remove support for older Chef Infra clients before 17, this cookbook needs some stuff to remain backwards compatible.

Reply:
Also, v4.0.0 of the line cookbook, which this change also updated to pull in, does not address the deprecated issues. This cookbook should not have to wait for another cookbook to fix their failings in order to be released.